doc/architecture/sensor_view_configuration.adoc

```diff
@@ -0,0 +1,17 @@
+= Sensor view configuration
+
+The sensor view configuration configures the sensor view.
+Sensor-view-configuration data enables the environment simulation to supply the necessary input to a sensor model.
+
+There are two sources for sensor-view-configuration data.
+
+Sensor-view-configuration data can be provided by the environment simulation.
+Note that the data is intended for the automatic configuration of an environment simulation.
+Sensor-view-configuration data is not intended to provide parameters to a sensor model. 
```

Comment:
What is the intention then? Shouldn't it be included then directly in SensorView?